<ui version="4.0" >
 <class>ChaserEditor</class>
 <widget class="QDialog" name="ChaserEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Chaser editor</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="m_nameLabel" >
       <property name="text" >
        <string>Chaser name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_nameEdit" >
       <property name="toolTip" >
        <string>Name of the chaser being edited</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QTreeWidget" name="m_tree" >
       <property name="rootIsDecorated" >
        <bool>false</bool>
       </property>
       <property name="itemsExpandable" >
        <bool>false</bool>
       </property>
       <property name="allColumnsShowFocus" >
        <bool>true</bool>
       </property>
       <column>
        <property name="text" >
         <string>Step</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Function</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QToolButton" name="m_add" >
         <property name="toolTip" >
          <string>Add step(s) to the current position</string>
         </property>
         <property name="text" >
          <string>...</string>
         </property>
         <property name="icon" >
          <iconset resource="main.qrc" >
           <normaloff>:/edit_add.png</normaloff>:/edit_add.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>26</width>
           <height>26</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_remove" >
         <property name="toolTip" >
          <string>Remove the selected step</string>
         </property>
         <property name="text" >
          <string>...</string>
         </property>
         <property name="icon" >
          <iconset resource="main.qrc" >
           <normaloff>:/edit_remove.png</normaloff>:/edit_remove.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>26</width>
           <height>26</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_raise" >
         <property name="toolTip" >
          <string>Raise the selected step once</string>
         </property>
         <property name="text" >
          <string>...</string>
         </property>
         <property name="icon" >
          <iconset resource="main.qrc" >
           <normaloff>:/up.png</normaloff>:/up.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>26</width>
           <height>26</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_lower" >
         <property name="toolTip" >
          <string>Lower the selected step once</string>
         </property>
         <property name="text" >
          <string>...</string>
         </property>
         <property name="icon" >
          <iconset resource="main.qrc" >
           <normaloff>:/down.png</normaloff>:/down.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>26</width>
           <height>26</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Run Order</string>
       </property>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QRadioButton" name="m_loop" >
          <property name="toolTip" >
           <string>Run through over and over again</string>
          </property>
          <property name="text" >
           <string>Loop</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_singleShot" >
          <property name="toolTip" >
           <string>Run through once and stop</string>
          </property>
          <property name="text" >
           <string>Single Shot</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_pingPong" >
          <property name="toolTip" >
           <string>First run forwards, then backwards, again forwards, etc.</string>
          </property>
          <property name="text" >
           <string>Ping Pong</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="title" >
        <string>Direction</string>
       </property>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QRadioButton" name="m_forward" >
          <property name="toolTip" >
           <string>Start from the first step</string>
          </property>
          <property name="text" >
           <string>Forward</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_backward" >
          <property name="toolTip" >
           <string>Start from the last step</string>
          </property>
          <property name="text" >
           <string>Backward</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="main.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ChaserEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChaserEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
